source/SIL.AppBuilder.Portal/src/auth.ts

```diff
@@ -2,6 +2,7 @@ import { SvelteKitAuth, type DefaultSession, type SvelteKitAuthConfig } from '@a
 import Auth0Provider from '@auth/sveltekit/providers/auth0';
 import { redirect, type Handle } from '@sveltejs/kit';
 import { DatabaseWrites, prisma } from 'sil.appbuilder.portal.common';
+import { RoleId } from 'sil.appbuilder.portal.common/prisma';
```

Comment:
I have no clue. That was something @7dev7urandom had committed at some point and it ended up in my branch when we did some rebasing to add in some bugs he fixed so that I could run the S1 UI

Reply:
When I view in VSCode, it is an unused import.  Try to remove it and see if everything is ok.

Reply:
OK. We will leave it for now...